pandas/core/generic.py

```diff
@@ -842,7 +842,25 @@ def __contains__(self, key):
 
     @property
     def empty(self):
-        """True if NDFrame is entirely empty [no items]"""
+        """True if NDFrame is entirely empty [no items], i.e. any of the axes
+        are of length 0.
+
+        Notes
+        -----
+        If NDFrame contains only NaNs, it is still not considered empty.
+
+        Examples
+        --------
+
+        >>> # containing only NaNs does not make the df empty
```

Comment:
Can you make this a normal sentence (not a comment in the code block), and then leave a blank line between that and the code block on the following lines ?